### DISCUSSION TITLE
Error message

### DISCUSSION THREAD

**Discussion by akariwrynn** [pybamm-team/PyBaMM#2307] [General]
Hi,
Sorry to asking so many questions but I couldn't figure out why I'm getting this error "WARNING] processed_variable.__call__(485): Calling variable outside interpolation range (returns 'nan')" when I changed the size of the battery which is too big (1.58m to 0.65m) as default or the electrolyte conc. How can I fix this issue or why this is happening?
Thanks and take care.

**Comment by valentinsulzer** [comment]
Please post full script as a [MWE](https://stackoverflow.com/help/minimal-reproducible-example)
